staff/bilal-elbedoui/skylab-semantics/README.md

```diff
@@ -0,0 +1,139 @@
+# Skylab semantics
```

Comment:
@bilalelbedoui93 please remove this file from this PR as it does not belong to this feature. Thanks!